QualityControl/Dockerfile

```diff
@@ -22,6 +22,7 @@ WORKDIR /usr/src/app
 # Install dependencies
 COPY package*.json ./
 RUN npm ci
+RUN npm i -g nodemon --save-dev
 
 
 # Copy project files
```

Comment:
This copy command will also copy `node_modules` if the user has any on the local setup and will override the freshly installed node_modules from L23.

Thus, i would recommend you first do command `COPY . .` . This will also copy package.json and package-lock.json. Then you do `npm ci --silent`